io/snmp/snmp.html

```diff
@@ -57,6 +57,58 @@
     });
 </script>
 
+<script type="text/x-red" data-template-name="snmp set">
+    <div class="form-row">
+         <label for="node-input-host"><i class="fa fa-globe"></i> Host</label>
+         <input type="text" id="node-input-host" placeholder="localhost">
+    </div>
+    <div class="form-row">
+        <label for="node-input-community"><i class="fa fa-user"></i> Community</label>
+        <input type="text" id="node-input-community" placeholder="public">
+    </div>
+    <div class="form-row">
+        <label for="node-input-version"><i class="fa fa-bookmark"></i> Version</label>
+        <select type="text" id="node-input-version" style="width: 150px;">
+            <option value="1">v1</option>
+            <option value="2c">v2c</option>
+        </select>
+    </div>
+    <div class="form-row">
```

Comment:
Maybe looking at the approach used in the switch node would help (specifically the rules section) [html](https://github.com/node-red/node-red/blob/master/nodes/core/logic/10-switch.html) and [js](https://github.com/node-red/node-red/blob/master/nodes/core/logic/10-switch.js).